packages/twenty-front/src/modules/settings/integrations/components/SettingsIntegrationDatabaseTablesListCard.tsx

```diff
@@ -25,27 +29,63 @@ const StyledRowRightContainer = styled.div`
 `;
 
 export const SettingsIntegrationDatabaseTablesListCard = ({
+  connectionId,
   tables,
 }: SettingsIntegrationDatabaseTablesListCardProps) => {
-  const { control } =
-    useFormContext<SettingsIntegrationsDatabaseTablesFormValues>();
+  const { syncRemoteTable } = useSyncRemoteTable();
+  const { unsyncRemoteTable } = useUnsyncRemoteTable();
 
+  // We need to use a state because the table status update re-render the whole list of toggles
+  const [items] = useState(
+    tables.map((table) => ({
+      id: table.name,
+      ...table,
+    })),
+  );
+
+  const onSyncUpdate = useCallback(
+    async (value: boolean, tableName: string) => {
```

Comment:
value naming is not clear
syncStatus?

Reply:
isSynced?

my brain had a hard time understanding if (value) line

Reply:
isSynced 👍